lncfg/wtclient.go

```diff
@@ -17,6 +17,11 @@ type WtClient struct {
 	// SweepFeeRate specifies the fee rate in sat/byte to be used when
 	// constructing justice transactions sent to the tower.
 	SweepFeeRate uint64 `long:"sweep-fee-rate" description:"Specifies the fee rate in sat/byte to be used when constructing justice transactions sent to the watchtower."`
+
+	// SessionCloseRange is the range over which to choose a random number
+	// of blocks to wait after the last channel of a session is closed
+	// before sending the DeleteSession message to the tower server.
+	SessionCloseRange uint32 `long:"session-close-range" description:"The range over which to choose a random number of blocks to wait after the last channel of a session is closed before sending the DeleteSession message to the tower server. Set to 1 for no delay."`
```

Comment:
This is to be reorg safe?

Reply:
discussed offline but mentioning here for other reviewers:

It is not about re-org safety but is rather for privacy. We dont necessarily want to tell the tower that it can delete a session _as soon as_ all the associated channels are closed cause then it will be able to do some chain-analysis things to figure out which channels you own. So this parameter lets the user choose a range from which a numbe-number-of-blocks-to-wait-after-channel-closure will be randomly chosen.